guide.md

```diff
@@ -1923,7 +1923,7 @@ It will also be crucial not to power on that burner phone ever (not even without
 
 See [Appendix N: Warning about smartphones and smart devices]
 
-You should test that the phone is in working order before going to the next step. ButWewill repeat myself and state that it is important to leave your smartphone at home when going (or turn it off before leaving if you must keep it) and that you test the phone at a random location that cannot be tracked back to you (and again, do not do that in front of a CCTV, avoid cameras, be aware of your surroundings). No need for Wi-Fi at this place either.
+You should test that the phone is in working order before going to the next step. But we will repeat myself and state that it is important to leave your smartphone at home when going (or turn it off before leaving if you must keep it) and that you test the phone at a random location that cannot be tracked back to you (and again, do not do that in front of a CCTV, avoid cameras, be aware of your surroundings). No need for Wi-Fi at this place either.

```

Comment:
"But I will" repeat myself, read the contributing guidelines about find/replace.